PNPM 8 new release breaking change in lock file definition prevent building docker container

**API Platform version(s) affected**: 3.0.9 (and probably all previous versions using pnpm 7)
<!-- Warning: if your version is 2.7 we ask you to try the 3.0 version. If the bug is present there we will fix it. If the bug is due to old 2.6 behavior we can't fix it.
In that last case we recommend to either upgrade to 3.0 or to temporarily replace concerned service with your own. -->

**Description**  
<!-- A clear and concise description of the problem. -->
This error show up when building container
`ERR_PNPM_FROZEN_LOCKFILE_WITH_OUTDATED_LOCKFILE  Cannot perform a frozen installation because the version of the lockfile is incompatible with this version of pnpm`

**How to reproduce**  
<!-- Code and/or config needed to reproduce the problem. If it's a complex bug,
     create a dedicated GitHub repository. -->
Download latest api platform release
Launch `docker compose up -d`

**Possible Solution**  
<!--- Optional: only if you have suggestions on a fix/reason for the bug. -->
In PWA Dockerfile fix pnpm version to 7 by replacing 
`pnpm@latest` by `pnpm@latest-7`

**Additional Context**  
<!-- Optional: any other context about the problem: log messages, screenshots, etc. -->
<img width="962" alt="image" src="https://user-images.githubusercontent.com/1615956/229547175-ff93758b-523f-4602-8ab3-c0589aac93d9.png">

--- a/pwa/Dockerfile
+++ b/pwa/Dockerfile
@@ -8,7 +8,7 @@ RUN apk add --no-cache libc6-compat
 WORKDIR /srv/app
 
 RUN corepack enable && \
-	corepack prepare --activate pnpm@latest && \
+	corepack prepare --activate pnpm@latest-7 && \
 	pnpm config -g set store-dir /.pnpm-store
 
 # Next.js collects completely anonymous telemetry data about general usage.

